akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -190,6 +191,9 @@ class HttpExt(private val config: Config)(implicit val system: ActorSystem) exte
 
   // ** CLIENT ** //
 
+  private[this] val poolMasterActorRef = system.actorOf(Props(new PoolMasterActor).withDeploy(Deploy.local), "pool-master")
+  private[this] val systemMaterializer = ActorMaterializer()
```

Comment:
This touches the somewhere else touched upon topic of 'then we should allow configuring it',
but we can tackle that along the way from here on